app/Server/src/main/java/Servlets/SkierServlet.java

```diff
@@ -166,12 +172,13 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
             Jedis jedis = jedisPool.getResource();
             String skierId = urlParts[1];
             String skiersKey = String.format("skiers/%s", skierId);
-            String resortId = req.getParameter("resort");
-            if (resortId == null) {
-                resp.setStatus(HttpServletResponse.SC_BAD_REQUEST);
-                resp.getWriter().write("{\"message\": \"Invalid inouts supplied\"}");
-                return;
-            }
+            String resortId = "1";
```

Comment:
Could you please get resortID and seasonID from either URL parts or request parameters?